### DISCUSSION TITLE
Richiesta uscita anticipata

### DISCUSSION THREAD

**Discussion by Forrest-It** [iisgiua/giuaschool#427] [Richieste di aiuto]
Buon giorno, 

Volevo chiedere se è possibile variare/disabilitare l'avviso/blocco che permette di richiedere un uscita anticipata non oltre le ore 16 del giorno prima. Grazie

**Comment by trinko** [comment]
Per configurare l’ora entro cui inviare le richieste, modificare il parametro di configurazione **scadenza_invio_richiesta**.

Ad esempio, se vuoi permettere che vengano inviate il giorno prima, senza limiti di orario, inserisci "23:59" come scadenza.

**Reply by Forrest-It** [reply]
Grazie
